python/src/kulya_python/utils.py

```diff
@@ -0,0 +1,62 @@
+import json
+import os
+from importlib.resources import files
+from .node import Node
+
+READ_JSON_NAME = "info.json"
+TARGET_JSON_FILE = "data.json"
+DEVELOPMENT_MODE = False
+
+"""
+A function that consumes all the files in "_data" and create a single JSON file with the data and the corresponding path.
+"""
+def directoryToOneJsonFile(path: str):
+    try:
+        targetFile = open(TARGET_JSON_FILE, "w", encoding="utf-8")
+        targetFile.write("{")
+        for folder_name, subfolders, filenames in os.walk(path):
+            for filename in filenames:
+                if filename == READ_JSON_NAME:
+                    file = open(folder_name + "\\" + filename, "r", encoding="utf-8")
+                    formatted_path = (
+                        folder_name.split("_data")[1].replace("\\", "/").strip("/")
+                    )
+                    targetFile.write('"' + formatted_path + '":')
+                    targetFile.write(file.read())
+                    targetFile.write(",")
+                    file.close()
+        targetFile.write('"":{}')
```

Comment:
writing the json format by hand is error prune, maybe we can use a more rebust solution by using the existing json module, where:
- we would have a local jsonData variable as an empty array, that we then add nodes into
- then we json.dump it to targetFile

this way we guarantee we have a valid and escaped json file, and as a bonus we have a nice formatted file ;)